### ISSUE TITLE
Maybe add render polygons?

### ISSUE THREAD

**Issue by continue98** [RIDE-2DAY/GZ_Shapes#2]
Hi, my friend. I want to suggest drawing polygons (dots). Closed to each other. Something like this: https://github.com/Seregamil/SA-MP-Polygons

But it does not work very well. Rendrites are not beautiful. Maybe you want to add polygons, and also draw a square for 2 points (I can help you with the formula for this, but I think you know how to do it yourself)

I hope that you will add the above to me. Thank you for attention

**Comment by Vasily-X** [comment]
Hello @continue98, by polygons you mean a square for each point you provide on that array or drawing a line between the points provided?

May you attach a screenshot?

Regards.

**Comment by continue98** [comment]
Yes, draw line of float array point. And it is desirable to make the construction of a square for 2 points. Most likely this will work:

v=+-(A2y-A1y, A1x-A2x)
A3=A2+v
A4=A1+v

where v - a vector

Example (polygons). This is not a square, it needs to be built from more than 2 points. I hope you understand the point. [[photo]](https://u.teknik.io/cqgLm.png)

**Comment by ziggi** [comment]
Not compatible with the name of the library :). But I think this should be added anyway.

**Comment by Vasily-X** [comment]
@ziggi true, I should change the name to `GZ Shapes` or similar. Whenever I get some free time I'll work on this.
